docs/usage.md

```diff
@@ -9,6 +9,9 @@ The CICD Configuration Wizard widget helps set up source control and streamline
 
 - `Define Environment` - Helps choose the FortiSOAR environment type, from development and production, as per your requirements 
 
-- `Connect Version Control` - Configures GitHub version control to perform Continous Delivery operations
+- `Define Your Source Control` - Select the source control you want to configure and provide the base url of the source control.
```

Comment:
This is now "Select Source Control" @cs-abhishek-shukla

Reply:
done